docs/source/programming_resources/tutorial_specific/android_studio/downloading_as_project_folder/Downloading-the-Android-Studio-Project-Folder.rst

```diff
@@ -1,5 +1,5 @@
-Downloading the Android Studio Project Folder
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Downloading the Android Studio Project Folder :bdg-success:`AS` :bdg-warning:`Legacy`
+=======================================================================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".